### ISSUE TITLE
Format for printing plate definition

### ISSUE THREAD

**Issue by Srceh** [IRC-SPHERE/HyperStream#25]
According to the example in plate_manager.py, the format to read plate definitions should be:

 {'H1': [(('house', '1'),)]}

that is: **_identifier[tag:data]_**.

However, for the current implementation, it is: **_tag[identifier:data]_**. 

( in container.py ('{0}[{1}:{2}]'.format(display_value, self[nid].identifier, str(self[nid].data))))

So I just wonder if  the previous one makes more sense.

For instance, for tutorial 5:

country[country_NZ:NZ]
country[country_Australia:Australia]

We can change it to:

country_NZ[country:NZ]
country_Australia[country:Australia]

**Comment by Srceh** [comment]
or [tag:data]identifier

It is just given (identifier = tag + data), I don't feel right to put identifier and data together in a [].

**Comment by tdiethe** [comment]
I guess you're talking about the output that is printed by a `MetaDataTree` object? It's worth pointing out the history here: I wanted to use a tree structure, and treelib was nearly an ideal off-the-shelf implementation, but I did find the back end printing wasn't great, so I created the MetaDataTree object which only overrides that.

That said, I actually think your way of displaying it (country_NZ[country:NZ]) totally makes sense for the displaying of the tree. In that case I'm happy for that to be changed round.

Or did you (also) mean something else?
